### ISSUE TITLE
Re-adjust modulefiles in a way that is compatible for module whatis

### ISSUE THREAD

**Issue by fgeorgatos** [hpcugent/easybuild-framework#427]
What we are doing now is to supply the same docstring text content for both
- proc ModulesHelp { } { docstring }
- module-whatis { docstring }

The 2nd one should rather be short and intended as a oneliner description.

A potential approach could be to populate the whatis string with only the first line of the original docstring or, take the text until the first "." (end of 1st sentence).
Or both, which perhaps would make the approach the safest as regards not losing information.

Example:

```
# module whatis
 intel/11.1.073       : Set up environment for the Intel C/C++ compiler suite v11.1.073
 intel/12.3.174       : Set up environment for the Intel C/C++ compiler suite v12.3.174
 openmpi/1.4.2-intel  : Set up environment for openmpi-1.4.3 using intel compilers
 fftw/2.1.5           : Set up environment for fftw-2.1.5
 fftw/3.2.2           : Set up environment for fftw-3.2.2
 abinit/6.2.2         : Set up PATH for Abinit 6.2.2 using openmpi-1.4.3 and intel 11.1.073
 abinit/6.4.3         : Set up PATH for Abinit 6.4.3 using openmpi-1.4.3 and intel 11.1.073
 abinit/6.6.2         : Set up PATH for Abinit 6.6.2 using openmpi-1.4.3 and intel 11.1.073
 elk/1.2.20           : Set up PATH for Elk 1.2.20 using intel 11.1.073
 espresso/4.2.1       : Set up environment for espresso-4.2.1
 espresso/4.3         : Set up environment for espresso-4.3
 qbox/1.50.4          : Set up PATH for Qbox 1.50.4 using openmpi-1.4.1 and intel 11.1.069
 qbox/1.52.3          : Set up PATH for Qbox 1.52.3 using openmpi-1.4.3 and intel 11.1.073
 siesta/3.0-rc2       : Set up PATH for siesta-3.0-rc2
 siesta/3.0-trunk041411: Set up PATH for siesta-3.0-trunk041411
 mpi                  : Set up environment for OpenMPI-1.2.5
 mvapich2/1.2p1       : Set up environment for mvapich2-1.2p1 using intel compilers
 mvp                  : Set up environment for mvapich2 v1.0.2 w/ pvfs2
 openmpi/1.3.2-gcc    : Set up environment for openmpi-1.3.2 using gcc compilers
 openmpi/1.3.2-intel  : Set up environment for openmpi-1.3.2 using intel compilers
 openmpi/1.4-qlc-gcc  : Set up environment for openmpi-1.4-qlc using gcc compilers
```

Reporting the compilation toolchain in some way would bring extra bonus for this issue...
(though someone can claim that the easybuild provided suffix covers for that).

ps. 
watch out, there is also a known bug with 64b version of environment modules that may affect this.
IFF that is the case, we could provide an easyconfig for 32b env. modules (doh!).

**Comment by boegel** [comment]
@fgeorgatos: fixed with #1271?

**Comment by fgeorgatos** [comment]
@boegel: only seeing this now (>1yr l8r!). Yes, I'd consider this handled so closing.
